python/pyspark/serializers.py

```diff
@@ -64,6 +64,7 @@
     from itertools import izip as zip, imap as map
 else:
     import pickle
+    basestring = unicode = str
```

Comment:
@HyukjinKwon I thought we have other places for this kind of thing (or is it your new PR for cloudpickle)

Reply:
Yes ..  there are some places that use this here and there. IIRC, we discussed about Python 2 drop in dev mailing list. I could get rid of it soon anyway ..

Reply:
Yeah, this and below are just for Python 2 support. Are we dropping that for Spark 3.0?

Reply:
I don't think we'll drop it in Spark 3.0. I will cc you in the related PRs later in the future.